moviepy/video/VideoClip.py

```diff
@@ -326,7 +326,8 @@ def write_videofile(self, filename, fps=None, codec=None,
                            logger=logger)
 
         if remove_temp and make_audio:
-            os.remove(audiofile)
+            if os.path.exists(audiofile):
+                os.remove(audiofile)
```

Comment:
First off, thanks for the contribution!

Generally, this looks good to me.  Should we move the `os.path.exists()` to [`L328`](https://github.com/Zulko/moviepy/pull/958/files#diff-4b7e3f0e11cb6f470780d69172d8fbe1L328) though?  I suppose it's trivial, regardless.

Reply:
I think is not the same, in L328 u are asking if `remove_temp` parameter is given and is not the same as to ask if the temp file exists

Reply:
Either way, it makes no difference. Thanks for contributing it though!